quickstart.py

```diff
@@ -16,7 +16,6 @@
 from fidesops.models.policy import ActionType
 
 logger = logging.getLogger(__name__)
-logging.basicConfig(level=logging.INFO)
```

Comment:
Can you add this back?  Most of the quickstart is the logged output, without this, it doesn't display very much.

Reply:
This does seem necessary for the quickstart though (test with `make quickstart`) - I wonder if it's because when I do docker exec to run `python quickstart.py` on the already running fidesops container that it doesn't have the same access to the logging config that was set at the application level, that it's a separate process.
```
quickstart: compose-build
	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
	@docker exec -it fidesops python quickstart.py
```

Reply:
ohh you're totally right. I'll update this, thanks!